### DISCUSSION TITLE
Query Text filter conditions not working with Microsoft as data source

### DISCUSSION THREAD

**Discussion by Ami99049** [microsoft-search/pnp-modern-search#3228] [Q&A]
I am using Microsoft Search as a Data Source with People as Entity Type . I want to filter our results based on User Email Id but filter conditions are not working in Text filter. When I used Sharepoint as a Data source I am able to filter the results and Query Text is working fine. Is there any way to Filter data while using MS as a Data source.
Isit possible to show additional fields like mobile number.

**Comment by wobba** [comment]
Unfortunately there is little information on how to filter on "person" via the Graph Search API, and this is not the same as searching for people via the SharePoint provider and UPA. See the first paragraph at https://learn.microsoft.com/en-us/graph/search-concept-person which mention what data is included.
